app/views/container_service/_main.html.haml

```diff
@@ -3,6 +3,8 @@
   .col-md-12.col-lg-6
     = render :partial => "shared/summary/textual", :locals => {:title => _("Properties"), :items => textual_group_properties}
     = render :partial => "shared/summary/textual_multilabel", :locals => {:title => _("Port Configurations"), :items => textual_group_port_configs}
+    = render :partial => "shared/summary/textual", :locals => {:title => _("Labels"), :items => textual_container_labels}
+    = render :partial => "shared/summary/textual", :locals => {:title => _("Selector"), :items => textual_container_selectors}
```

Comment:
> I think it should be "Selectors" (plural)

Each entity has only one selector with multiple parts (as opposed to labels). So IMO 'Selectors' is confusing in this context.